site/app/templates/notebook/Notebook.twig

```diff
@@ -290,7 +290,7 @@
     //interval is 20 percent of overall time allowed or 10 minutes, whichever is greater
     let interval = Math.floor(((({{ allowed_minutes }}) / 5) > 10 ? ({{ allowed_minutes }}) / 5 : 10));
     let loadTime = 0;
-    let showPopup = false;
+    let show_popup = false;
```

Comment:
Why was this change necessary?  If you're fixing a bug these tests uncovered, would it be better to make a separate bugfix PR and then update this PR once that gets merged?

Reply:
reverted the changes